.gitpod.yml

```diff
@@ -2,6 +2,7 @@ image:
   file: .gitpod.dockerfile
 ports:
 - port: 3000 # Trace viewer example application
+  onOpen: ignore
```

Comment:
This prevents the pop-up, that you can click to open the example application. I was wondering why it did not show.